packages/styles/scss/components/controls/_button.scss

```diff
@@ -25,6 +24,11 @@
     }
   }
 
+  &:not([disabled])[data-active='true'] {
+    color: var(--accent-fg);
+    background-color: var(--accent-bg);
```

Comment:
I think it is a bit unexpected that this does not use `var(--control-active-bg)`. If we want to keep the blue accent color (I don't mind if we use the default colors control btn colors), we should probably overwrite it in `_chat-toggle.scss`. 
If we decide to use aria-pressed instead of `data-lk-active`, the whole rule might be redundant because of the rule on line 19.